strawberry_django_plus/mutations/resolvers.py

```diff
@@ -290,8 +291,12 @@ def update(info, instance, data, *, full_clean=True):
         for file_field, value in files:
             file_field.save_form_data(instance, value)
 
-        if full_clean:
+        if isinstance(full_clean, bool) and full_clean:
             instance.full_clean()
+        elif isinstance(full_clean, dict):
+            instance.full_clean(**FullClean(**full_clean).__dict__)
+        elif isinstance(full_clean, FullClean):
+            instance.full_clean(**full_clean.__dict__)
```

Comment:
Instead of using a dataclass here, maybe use a `TypedDict` to type the options and use the dict directly?

The advantage is that there's nothing that needs to be imported, specially because it is just an object used for the parameter itself, and it is still properly typed